### DISCUSSION TITLE
Actual track count in album

### DISCUSSION THREAD

**Discussion by JOJ0** [beetbox/beets#4669] [Q&A] [answered]
It's probably a dumb question but what is the name of $field to find out how many tracks are actually containend in an album in the library.

I know there is `$albumtotal`, which is an album level field. I can access it like that for example:

```
format_album: "%if{$albumartist,$albumartist,$artist} - $original_year [$catalognum] - $album ($albumtotal)
```

It gives me the count the metadata source I used to tag the album had provided, but if there are missing tracks how do I find out how many I definitely have? Do I really need to do:

```
album_fields:
    wehave: return len(items)
```

or is there a built-in field already?

**Comment by sampsyo** [comment, marked as answer]
We don't have anything built-in quite like that—indeed, `$albumtotal` has more complicated logic to use the _metadata's_ indication of the number of tracks rather than the _actual_ number of associated items. But the `inline` configuration you have mentioned seems like it might be what you need!

**Reply by JOJ0** [reply]
Alright, thanks so much for clarifying! Perfect :-)
